src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -133,7 +133,7 @@ public static Item getItemFromBlock(final Block block)
     @NotNull
     public static Boolean isItemStackEmpty(@Nullable final ItemStack stack)
     {
-        return stack == null || stack == EMPTY || stack.getCount() <= 0;
+        return stack == null || stack == EMPTY || stack.getItem() != null || stack.getCount() <= 0;
```

Comment:
I'm not really sure if that could still happen if porting a world from 1.10 to 1.11.
If not, we only need it in 1.10 then.